packages/@aws-cdk/service-spec-importers/src/importers/import-cloudformation-registry.ts

```diff
@@ -105,7 +105,11 @@ export function importCloudFormationRegistryResource(options: LoadCloudFormation
 
       if (jsonschema.isAnyType(resolvedSchema)) {
         return { type: 'json' };
-      } else if (jsonschema.isOneOf(resolvedSchema) || jsonschema.isAnyOf(resolvedSchema)) {
+      } else if (
+        jsonschema.isOneOf(resolvedSchema) ||
+        jsonschema.isAnyOf(resolvedSchema) ||
+        jsonschema.isAllOf(resolvedSchema)
```

Comment:
what is the impact of adding AllOf here. I believe allOf should be handled in a different way, as I believe it should be the union of all types' properties